controller/plugins/actuator/builder.py

```diff
@@ -120,12 +119,10 @@ def get_actuator(self, name, parameters={}):
             return KVMUPVActuator(iops_reference, bs_reference)
 
         elif name == "k8s_replicas":
-            try:
-                actuator = K8sActuator(parameters['app_id'], 
-                               api.k8s_manifest)          
-            except Exception as e:
-                raise e
-            
+            actuator = K8sActuator(parameters['app_id'],
```

Comment:
Removed try/except from here since the only exception possible here is when Kubernetes API can't load the config file. This situation is caused by bad configuration so there's nothing to do but logging it.